addons/account_analytic_default/models/account_analytic_default.py

```diff
@@ -63,6 +63,27 @@ def account_get(self, product_id=None, partner_id=None, user_id=None, date=None,
 class AccountInvoiceLine(models.Model):
     _inherit = "account.invoice.line"
 
+    def default_get(self, fields_list):
```

Comment:
```suggestion
   @api.model
    def default_get(self, fields_list):
```

Reply:
whoops! fixed in 2c0a2b2bea5de1d3f330dbb4ec18057280bb33d1